handbook/ops/bizops/user_definitions.md

```diff
@@ -60,22 +60,30 @@ In the site admin panel, we have a Usage stats page that displays number of MAUs
 
 ## Sourcegraph Cloud
 
-### Funnel definitions
+### User status definitions
 
-| Status      | Description                                                                |
-| ----------- | -------------------------------------------------------------------------- |
-| Visitor     | A user who landed on Sourcegraph Cloud                                     |
-| Active      | A user who performed an activation qualifying event once in the timeperiod |
-| Retained    | A user who were active last month _and_ this month                         |
-| Churned     | A user who were active last month but not this month                       |
-| Resurrected | A user who were _not_ active last month but are active this month          |
-| Registered  | A user who created an account on Sourcegraph Cloud                         |
+| Status      | Description                                                                                            |
+| ----------- | ------------------------------------------------------------------------------------------------------ |
+| Visitor     | A user who landed on Sourcegraph Cloud                                                                 |
+| Active      | A user who performed an active event for the time period (see Amplitude for the list of active events) |
+| Retained    | A user who were active last month _and_ this month                                                     |
+| Churned     | A user who were active last month but not this month                                                   |
+| Resurrected | A user who were _not_ active last month but is active this month                                       |
+| Registered  | A user who created an account on Sourcegraph Cloud                                                     |
 
-##### Activation qualifying events
+### Key metrics
 
-Users activate if they perform one of these qualifying events:
+We track the following key metrics:
 
-| Product area              | Qualifying events                  | Description                                                                                         |
+- acquisition: number of visitors
+- activation: the percentage of users that had a successfull initial experience.
+- retention: the number of active users retained time period over time period
+
+#### Activation
```

Comment:
Just to confirm, the list of events below is not the same as the list of events that is used as the definition of an active user in a period, right? That list has more included (such as viewing search results)?

Reply:
Indeed. The list of events is very large, and define in Amplitude. This is defined in the table above.